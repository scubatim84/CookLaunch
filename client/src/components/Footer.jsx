import React from 'react';
import Container from '@material-ui/core/Container';

import Copyright from './Copyright';
import useStylesMain from '../Styles';
import { themeMain } from '../Theme';

const Footer = () => {
  const classes = useStylesMain(themeMain);

  return (
    <footer className={classes.footer}>
      <Container maxWidth="xs">
        <Copyright />
      </Container>
    </footer>
  );
};

export default Footer;
