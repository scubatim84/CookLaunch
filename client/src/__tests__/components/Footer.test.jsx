import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../../components/Footer';

describe('Footer', () => {
  it('Renders component without crashing', () => {
    render(<Footer />);
  });
});
