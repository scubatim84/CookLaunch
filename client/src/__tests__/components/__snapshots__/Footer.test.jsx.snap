import React from 'react';
import {shallow} from 'enzyme';
import Footer from '../../components/Footer';

describe('Footer', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
